client/router.js

```diff
@@ -302,6 +307,18 @@ Router.prototype._notfoundRoute = function(context) {
 Router.prototype.initialize = function() {
   var self = this;
   this._updateCallbacks();
+
+  var oldShow = this._page.show;
+
+  this._page.show = function(path, state, dispatch, push) {
```

Comment:
I think we need more details on what's going on here.
So, it's clear on the future too.